backend/src/main/java/com/github/boltzmann/biokiste/backend/service/exceptions/NoSuchOrganicBoxException.java

```diff
@@ -0,0 +1,9 @@
+package com.github.boltzmann.biokiste.backend.service.exceptions;
+
+import java.util.NoSuchElementException;
+
+public class NoSuchOrganicBoxException extends NoSuchElementException {
+    public NoSuchOrganicBoxException(String s) {
+        super(s);
+    }
```

Comment:
see above. It's basicly the same with another name?

Reply:
Yes.